docs/themes/porter/assets/sass/docs-home.scss

```diff
@@ -93,7 +124,7 @@
 
       &:after {
         $height: 6px;
-        background: url('/img/pattern-waves.png') left top repeat-x;
+        // background: url('/img/pattern-waves.png') left top repeat-x;
```

Comment:
Can you remove any commented out lines like this one here? I saw a few others that can be removed as well.